build.sbt

```diff
@@ -32,12 +31,16 @@ lazy val codegen = Project(id = akkaGrpcCodegenId, base = file("codegen"))
     mainClass in assembly := Some("akka.grpc.gen.Main"),
     assemblyOption in assembly := (assemblyOption in assembly).value.copy(prependShellScript =
         Some(sbtassembly.AssemblyPlugin.defaultUniversalScript(shebang = true))),
-    crossScalaVersions -= scala213))
+    crossScalaVersions := Dependencies.Versions.CrossScalaForPlugin,
+    scalaVersion := scala212))
```

Comment:
I think `scalaVersion := scala212` looks nicer to me here - after all, we want it to explicitly be 2.12 since that's what sbt expects. For projects that aren't (part of) an sbt plugin indeed `crossScalaVersions.head` seems nicer